src/libraries/System.Private.CoreLib/src/ILLink/ILLink.Substitutions.Browser.xml

```diff
@@ -1,7 +1,10 @@
 <linker>
   <assembly fullname="System.Private.CoreLib">
-    <type fullname="System.Globalization.GlobalizationMode">
-      <method signature="System.Boolean get_Hybrid()" body="stub" value="true" feature="System.Globalization.Hybrid" featurevalue="false" />
+      <type fullname="System.Globalization.GlobalizationMode">
+      <method signature="System.Boolean get_Hybrid()" body="stub" value="true" feature="System.Globalization.Hybrid" featurevalue="true" />
+    </type>
+    <type fullname="System.Globalization.GlobalizationMode/Settings">
+      <method signature="System.Boolean get_Hybrid()" body="stub" value="false" feature="System.Globalization.Hybrid" featurevalue="false" />
```

Comment:
This looks weird:
* We have two properties to allow for better trimming, but functionally they should be the same I assume. So why do they have different substitutions?
* I think trimmer should be able to perform constant propagation across the `GlobalizationMode.Hybrid` property, meaning it should only be necessary to substitute the `GlobalizationMode.Settings.Hybrid` property, would need to be tested though.

Reply:
You are right, if we move both stubs to the Settings level then the result is the same.

Reply:
My mistake: both should be replaced on the `GlobalizationMode` level, not `GlobalizationMode.Settings` level. This is checked by `InvariantGlobalizationTrue` test and this kind of trimming allows us to remove the whole `Settings` class on Unix.